src/pkg/caendr/caendr/models/datastore/user_token.py

```diff
@@ -0,0 +1,29 @@
+from caendr.models.datastore import Entity
+
+
+class UserToken(Entity):
```

Comment:
consider renaming to `UserRevokedToken`

Reply:
I'm going to leave the current name for now since not all tokens of that DS kind are revoked.